tests/langchain/test_langchain_model_export.py

```diff
@@ -2465,23 +2468,23 @@ def test_save_load_chain_as_code(chain_model_signature, chain_path, model_config
     answer = "Databricks"
     assert loaded_model.invoke(input_example) == answer
     pyfunc_loaded_model = mlflow.pyfunc.load_model(model_info.model_uri)
-    assert (
-        pyfunc_loaded_model.predict(input_example)[0]
-        .get("choices")[0]
-        .get("message")
-        .get("content")
-        == answer
-    )
+    assert answer == _get_message_content(pyfunc_loaded_model.predict(input_example))
 
     response = pyfunc_serve_and_score_model(
         model_info.model_uri,
         data=json.dumps({"inputs": input_example}),
         content_type=pyfunc_scoring_server.CONTENT_TYPE_JSON,
         extra_args=["--env-manager", "local"],
     )
-    assert PredictionsResponse.from_json(response.content.decode("utf-8")) == {
-        "predictions": [try_transform_response_to_chat_format(answer)]
-    }
+    predictions = PredictionsResponse.from_json(response.content.decode("utf-8"))["predictions"]
+    expected = [try_transform_response_to_chat_format(answer)]
+
+    # Excldue the `created` timestamp from the comparison as it is not deterministic
+    def _pop_created_timestamp(predictions):
+        for prediction in predictions:
+            prediction.pop("created")
```

Comment:
not a blocker, but another option is to use `unittest.mock.ANY`: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.ANY

Reply:
Awesome idea!! It tells the intention more clearly, updated:)